web/src/components/CreateAssertionModal/newSelectorLogic.tsx

```diff
@@ -0,0 +1,20 @@
+import {IItemSelector} from '../../types/Assertion.types';
+
+export function newSelectorLogic(selectorList: IItemSelector[]): string {
+  let result = '';
+  for (let i = 0; i < selectorList.length; i += 1) {
+    const selector = selectorList[i];
+    result += `span[`;
+    result += selector.propertyName;
+    result += selector.operator ? ` ${selector.operator.toLowerCase()} ` : '=';
+    // add quotes if value is a string
+    if (selector.valueType === 'stringValue') result += `'`;
```

Comment:
I think we should have a more complex way of creating this chunk to support the other value types.